aderyn_core/src/detect/high/send_ether_no_checks.rs

```diff
@@ -24,11 +24,11 @@ pub struct SendEtherNoChecksDetector {
 impl IssueDetector for SendEtherNoChecksDetector {
     fn detect(&mut self, context: &WorkspaceContext) -> Result<bool, Box<dyn Error>> {
         for func in helpers::get_implemented_external_and_public_functions(context) {
-            let mut tracker = MsgSenderAndCallWithValueTracker::default();
+            let mut tracker = AddressChecksAndCallWithValueTracker::default();
             let callgraph = CallGraph::new(context, &[&(func.into())], CallGraphDirection::Inward)?;
             callgraph.accept(context, &mut tracker)?;
 
-            if tracker.sends_native_eth && !tracker.has_msg_sender_checks {
+            if tracker.sends_native_eth && !tracker.has_binary_checks_on_some_address {
                 capture!(self, context, func);
             }
         }
```

Comment:
@TilakMaddy can you please rewrite the title and description? They don't seem quite right.